### PR TITLE
fix: tips entrance ui / ux issues

### PR DESCRIPTION
**Pull request by BillySh1**
## Description

Closes #MF-596 MF-618 MF-650 MF-632

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

File: packages/mask/shared-ui/locales/en-US.json [hunk 1]
```diff
@@ -1088,7 +1088,7 @@
     "plugin_artblocks_share_no_official_account": "I just purchased a beautiful piece of art from '{{name}}' collection for {{price}} {{symbol}}. Welcome to join.",
     "plugin_tips_disconnect_introduction": "Are you going to disconnect {{persona}} from wallet <strong>{{address}}<strong/>?",
     "plugin_tips_disconnect_dialog_title": "Disconnect Wallet?",
-    "plugin_tips_not_evm_alert": "The tips feature only supports EVM chains, and other chains are starting to support it.",
+    "plugin_tips_not_evm_alert": "The tips feature currently supports only EVM chains. Support for other chains will be added in the future.",
```
Comment (UncleBill) [inline packages/mask/shared-ui/locales/en-US.json:1089]: Move to `plugins/Tips/src/locales`

File: packages/mask/src/plugins/Tips/hooks/useSupportedNetworks.ts [hunk 2]
```diff
@@ -1,13 +1,21 @@
...
+}
 
 export function useSupportedNetworks(keys: NetworkPluginID[]) {
     // todo support solana flow when NextID supported
-    const networkMap = {
-        [NetworkPluginID.PLUGIN_EVM]: { name: 'EVM Chian', icon: new URL('../assets/evmChains.png', import.meta.url) },
-        [NetworkPluginID.PLUGIN_FLOW]: { name: 'Solana Chain', icon: new URL('../assets/solana.png', import.meta.url) },
-        [NetworkPluginID.PLUGIN_SOLANA]: { name: 'Flow Chain', icon: new URL('../assets/flow.png', import.meta.url) },
+    const networkMap: Record<NetworkPluginID, SupportedNetworkMap> = {
+        [NetworkPluginID.PLUGIN_EVM]: { name: 'EVM Chian', icon: EVMChainsIcon },
```
Comment (UncleBill) [inline packages/mask/src/plugins/Tips/hooks/useSupportedNetworks.ts]:
```suggestion
        [NetworkPluginID.PLUGIN_EVM]: { name: 'EVM Chain', icon: EVMChainsIcon },
```

Comment (UncleBill) [reply packages/mask/src/plugins/Tips/hooks/useSupportedNetworks.ts]: You should export `networkMap` (maybe move to another file), and use it in `packages/mask/src/plugins/Tips/SNSAdaptor/components/WalletsByNetwork.tsx` for resolving network by network id.

File: packages/mask/src/plugins/Tips/SNSAdaptor/components/WalletsByNetwork.tsx [hunk 3]
```diff
@@ -39,10 +40,14 @@ const useStyles = makeStyles()((theme) => ({
...
     },
+    networkIcon: {
+        width: 'auto',
+        height: 18,
+    },
 }))
 
 interface WalletsByNetworkProps {
-    network: { name: string; icon: URL }
+    network: SupportedNetworkMap
```
Comment (UncleBill) [inline packages/mask/src/plugins/Tips/SNSAdaptor/components/WalletsByNetwork.tsx]: Pass `sufficient and necessary` data only, `networkId`, and resolve network by it.

**Review by UncleBill — CHANGES_REQUESTED** [review]

File: packages/mask/src/plugins/Tips/hooks/useSupportedNetworks.ts [hunk 4]
```diff
@@ -1,13 +1,21 @@
...
+        [NetworkPluginID.PLUGIN_FLOW]: { name: 'Solana Chain', icon: FlowIcon },
+        [NetworkPluginID.PLUGIN_SOLANA]: { name: 'Flow Chain', icon: SolanaIcon },
     }
-    return keys.reduce((res: Array<{ name: string; icon: URL }>, x) => {
+    return keys.reduce((res: Array<SupportedNetworkMap>, x) => {
```
Comment (septs) [inline packages/mask/src/plugins/Tips/hooks/useSupportedNetworks.ts]:
```suggestion
    return keys.reduce((res: SupportedNetworkMap[], x) => {
```

File: packages/mask/shared-ui/locales/en-US.json (hunk 1 above)
Comment (BillySh1) [reply packages/mask/shared-ui/locales/en-US.json:1089]: done

File: packages/mask/src/plugins/Tips/hooks/useSupportedNetworks.ts (hunk 2 above)
Comment (BillySh1) [reply packages/mask/src/plugins/Tips/hooks/useSupportedNetworks.ts]: done

File: packages/mask/src/plugins/Tips/SNSAdaptor/components/WalletsByNetwork.tsx (hunk 3 above)
Comment (BillySh1) [reply packages/mask/src/plugins/Tips/SNSAdaptor/components/WalletsByNetwork.tsx]: done

File: packages/mask/src/plugins/Tips/hooks/useSupportedNetworks.ts (hunk 4 above)
Comment (BillySh1) [reply packages/mask/src/plugins/Tips/hooks/useSupportedNetworks.ts]: done

File: packages/mask/src/plugins/Tips/locales/en-US.json [hunk 5]
```diff
@@ -25,5 +28,21 @@
...
+    "plugin_tips_persona_sign_error": "Persona Signature failed.",
+    "plugin_tips_wallet_sign_success": "The walelt is connected.",
+    "plugin_tips_wallet_sign_error": "Wallet connection failed.",
+    "plugin_tips_wallet_add": "Please add wallet",
+    "plugin_tips_connect_wallet": "Connect your wallet",
+    "plugin_tips_sns_persona_unmatched": "Twitter ID verified persona({{currentSNSConnectedPersonaPublicKey}}) is not consistent with your current persona({{currentPersonaPublicKey}}). Please switch to persona({{currentSNSConnectedPersonaPublicKey}}) and try again.",
+    "plugin_tips_verify_line_intro": "Please add and verify your wallet to set the address for receiving tips.",
+    "plugin_tips_empty_list": "Please set wallets to receive tips.",
+    "plugin_tips_set_as_default": "Set as default",
+    "plugin_tips_copy_success_of_wallet_addr": "Copy wallet address successfully!",
```
Comment (UncleBill) [inline packages/mask/src/plugins/Tips/locales/en-US.json]: `plugin_tips` can be omit.

Comment (BillySh1) [reply packages/mask/src/plugins/Tips/locales/en-US.json]: done

**Review by Jack-Works — DISMISSED** [review]

**Review by UncleBill — APPROVED** [review]

**Review by hanszhoou — APPROVED** [review]

**Pull request merged**